www/nginx/src/opnsense/service/templates/OPNsense/Nginx/nginx.conf

```diff
@@ -7,12 +7,20 @@ load_module /usr/local/libexec/nginx/ngx_http_brotli_static_module.so;
 load_module /usr/local/libexec/nginx/ngx_http_js_module.so;
 
 user www staff;
+{% if OPNsense.Nginx.http.workerprocesses is defined and OPNsense.Nginx.http.workerprocesses != '' %}
+worker_processes {{ OPNsense.Nginx.http.workerprocesses }};
+{% else %}
 worker_processes  1;
+{% endif %}
```

Comment:
since you made it required, it is always defined. Same applies to the worker connections.

Reply:
Do you think it is enough just to set worker_processes equal to OPNsense.Nginx.http.workerprocesses without the if/else? My concern is that a bug in the frontend could allow an empty set or null value to be set, which would cause nginx to crash.

Reply:
If you set it to required and change the model version,
the migration will set it to the default value and you cannot store it as empty because model validation will fail. The only case is a manipulated config.xml which has been uploaded.

The way you wrote it here is the requirement for required set to 'N' or omitted.